analyzers/tests/SonarAnalyzer.UnitTest/Json/LexicalAnalyzerTest.cs

```diff
@@ -90,6 +90,7 @@ public void ReadSpecialCharacters()
         [DataRow("6")]
         [DataRow("7")]
         [DataRow("8")]
+        [DataRow("9")]
```

Comment:
I did the MSIL math for `ReadSymbol()` method. Please add UTs that will start (and throw) with these characters:
```
#
$
%
&
'
(
)
*
+
.
/
```
That should be enough to get 96.26%

Reply:
Thanks for the math! I've pushed the corresponding commit